streams/src/test/java/org/apache/kafka/streams/processor/internals/ProcessorNodeTest.java

```diff
@@ -103,20 +103,14 @@ public void testMetrics() {
         final ProcessorNode<Object, Object> node = new ProcessorNode<>("name", new NoOpProcessor(), Collections.<String>emptySet());
         node.init(context);
 
-        final String[] latencyOperations = {"process", "punctuate", "create", "destroy"};
+        final String[] latencyOperations = {"process", "punctuate", "create"};
         final String throughputOperation = "forward";
         final String groupName = "stream-processor-node-metrics";
         final Map<String, String> metricTags = new LinkedHashMap<>();
         metricTags.put("processor-node-id", node.name());
         metricTags.put("task-id", context.taskId().toString());
         metricTags.put("client-id", "mock");
 
-
-        for (final String operation : latencyOperations) {
-            assertNotNull(metrics.getSensor("name-mock.0_0." + operation));
-        }
-        assertNotNull(metrics.getSensor("name-mock.0_0." + throughputOperation));
-
```

Comment:
We don't need to verify the sensors themselves, since we verify the metrics (which are the actual interface)